public/locales/dk/translation.json

```diff
@@ -100,6 +100,7 @@
         "tokenPairUnavailable": "",
         "retryOrCancel": "",
         "newSwap": "",
+        "selectedServer": "",
```

Comment:
When developing only edit `en/translation.json`. Translation team will take care of the other files. Also it will automatically fallback to english if undefined, so that's perfectly fine.

Reply:
Ok, will remove everything from non en files